editor/src/components/canvas/canvas-strategies/strategies/ancestor-metastrategy.tsx

```diff
@@ -39,16 +43,27 @@ export function ancestorMetaStrategy(
       return []
     }
 
+    const groupLikeChildren = retargetStrategyToChildrenOfContentAffectingElements(canvasState)
```

Comment:
Minor, but I think we should rename this, as it will only be the children if the original target was group-like. Maybe something as simple as `correctedTargets`?